docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -188,21 +182,22 @@ Accept: application/json
 <div class="HubBlock-content">
 
 > ### Note
-> * Note the widget's UUID in the response for creating the placement in the next step.
+> * Make note of the widget’s `uuid` in the response. You will use it to create a placement in the next step.
 
 </div>
 </div>
 </div>
 
 ## Create a placement
 
-In the UI, users drag and drop widgets to *place* them in a region on a page. Here, we'll use the API to programmatically place our widget. To do so, send a `POST` request to [`/v3/content/placements`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/placement/createplacement).
+In the control panel UI, users can drag and drop widgets to place them in a region on a page. For the purpose of this tutorial, we will use the Widgets API to place the widget programmatically. 

```

Comment:
In the control panel UI, users can drag and drop widgets to place them in a region on a page. For this tutorial, we will use the Widgets API to place the widget programmatically.